src/config.js

```diff
@@ -10,6 +10,8 @@ export default {
   outputDir: defaultOutputDir,
   allureOutputDir: 'allure-results',
 
+  screenshotPaddingWidth: 4,
```

Comment:
Let's document this new option somewhere

Reply:
I'll add comments to config to explain it.

But, currently this option is only accessible in the config file - it can't be overridden via the incoming session request options as I didn't think it made sense for it to work that way.

Should it be made so it can be modifiable like the other options?

Reply:
ahh this seems to be the ffmpg options. I think it makes sense to have this configurable as reporter option but we can tackle this in a separate PR.